### DISCUSSION TITLE
Why my filecoin `send` tx always failed with SysErrOutOfGas

### DISCUSSION THREAD

**Discussion by Ljzn** [filecoin-project/lotus#9695] [General Q&A] [answered]
This is my tx: https://filfox.info/en/message/bafy2bzacec7ibogqsfwmqf6n355tpmasoykif4edu3w3cfhmrpu73fyxfdt7q

The Gas config is 

```
Gas Fee Cap
9 nanoFIL

Gas Premium
151,270 attoFIL

Gas Limit
2,189,560

Base Fee
191,823 attoFIL
```

There is another `send` tx in the same block, that tx succeed: https://filfox.info/en/message/bafy2bzaceburugbfulh7gaaziwdzlkjcrojyyvfszauxnlaeljz475scwzfwm?block=bafy2bzacebxrauq7gxbitvqhon5t4fj27fzzv3tsg2cw66yqhp56ynjukvm76

I paid more gas fee then that tx, why my tx still failed?

**Comment by geoff-vball** [comment, marked as answer]
If you send a message to an address that doesn't exist yet, you will have to pay additional gas to create the new actor for this address. Sending a message to an account that already exists is much cheaper.

**Reply by Ljzn** [reply]
Thanks for your reply! How much does creating a new actor cost on average?

**Reply by geoff-vball** [reply]
I think its about 2M extra gas. If you send your message without specifying a gas limit, lotus should automatically calculate a reasonable gas limit for you.
